src/server/JasmineGraphInstanceService.cpp

```diff
@@ -3584,3 +3650,309 @@ map<long, map<long, unordered_set<long>>> calculateEgoNet(string graphID, string
+        string intermediate;
+        while (getline(wl, intermediate, ':')) {
+            workerSocketPair.push_back(intermediate);
+        }
+
+        Utils utils;
+        std::string aggregatorFilePath = utils.getJasmineGraphProperty("org.jasminegraph.server.instance.datafolder");
+        std::string centralGraphIdentifier = graphID + +"_centralstore_" + workerSocketPair[2];
+
+        std::string centralStoreFile = aggregatorFilePath + "/" + centralGraphIdentifier;
+        instance_logger.log("###INSTANCE### centralstore " + centralStoreFile, "info");
+
+        struct stat s;
+        if (stat(centralStoreFile.c_str(), &s) == 0) {
+            if (s.st_mode & S_IFREG) {
+                JasmineGraphHashMapCentralStore centralStore = JasmineGraphInstanceService::loadCentralStore(
+                        centralStoreFile);
+                map<long, unordered_set<long>> centralGraphMap = centralStore.getUnderlyingHashMap();
+
+                for (map<long, unordered_set<long>>::iterator centralGraphMapIterator = centralGraphMap.begin();
+                     centralGraphMapIterator != centralGraphMap.end(); ++centralGraphMapIterator) {
+                    long startVid = centralGraphMapIterator->first;
+                    unordered_set<long> endVidSet = centralGraphMapIterator->second;
+
+                    for (auto itr = endVidSet.begin(); itr != endVidSet.end(); ++itr) {
+                        map<long, long>::iterator fromDegreeDistIterator = fromDegreeDist.find(startVid);
+                        if (fromDegreeDistIterator != fromDegreeDist.end()) {
+                            long degree = fromDegreeDistIterator->second;
+                            fromDegreeDistIterator->second = degree + 1;
+                        } else {
+                            fromDegreeDist.insert(std::make_pair(startVid, 1));
+                        }
+
+                        if (graphVertexMap.find(*itr) != graphVertexMap.end()) {
+                            map<long, unordered_set<long>>::iterator toIDiterator = worldToLocalVertexMap.find(*itr);
+                            if (toIDiterator != worldToLocalVertexMap.end()) {
+                                unordered_set<long> fromIDs = toIDiterator->second;
+                                fromIDs.insert(startVid);
+                                toIDiterator->second = fromIDs;
+                            } else {
+                                unordered_set<long> fromIDs;
+                                fromIDs.insert(startVid);
+                                worldToLocalVertexMap.insert(std::make_pair(*itr, fromIDs));
+                            }
+                        }
+
+                    }
+                }
+            }
+        }
+    }
+
+    double alpha = 0.85;
+    double damp = 1 - alpha;
+    long entireGraphSize = atol(graphVertexCount.c_str());
+    float mu = (damp / entireGraphSize);
+
+
+    map<long, float> authResultMap;
+    float f = 0;
```

Comment:
Can we have better variable name than `f`?

Reply:
fixed

Reply:
Ok